.gitignore

```diff
@@ -1 +1,86 @@
 /vendor/
+composer.lock
```

Comment:
Compose lock should not be ignored

Reply:
Huh, I just realized `composer.lock` was never added to the original repo. Going to correct that now

Reply:
true that, my fault